Makefile

```diff
@@ -1,96 +1,34 @@
-.PHONY : test workspace archive unsafe_install install compile_commands debug run run_force test build build build-release
-
-ASSETDIR=XCHammerAssets
-ASPECTDIR=tulsi-aspects
+.PHONY : test workspace archive install compile_commands debug run run_force test build build build-release
 
 ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
-ifeq ($(BAZEL_BUILD),true)
-	PRODUCT=xchammer
-	XCHAMMER_BIN_BASE=$(ROOT_DIR)/xchammer.app/Contents/MacOS
-else
-	PRODUCT=XCHammer
-	XCHAMMER_BIN_BASE=$(PWD)/.build/debug
-endif
+PRODUCT=xchammer
+XCHAMMER_BIN_BASE=$(ROOT_DIR)/xchammer.app/Contents/MacOS
 
 XCHAMMER_BIN := $(XCHAMMER_BIN_BASE)/$(PRODUCT)
 
 PREFIX := /usr/local
 
-aspects:
-ifneq ($(BAZEL_BUILD),true)
-	# Export the tulsi workspace to PWD. We need this for
-	# Xcode, because there is no way to correctly install
-	# resources.
-	# Note that the build process always exports from this directory to the
-	# bundle
-	./export_tulsi_aspect_dir.sh ${PWD}/$(ASPECTDIR)
-endif
-
-generate_xcodeproj:
-	swift package generate-xcodeproj
-
-# Make a SPM generated Xcode project.
-#
-# Copy the tulsi-aspects and XCHammerAssets adjacent to the Xcode build
-# directory to allow loading of resources, since we can't express this in SPM
-#
-# Note: this is brittle and may not work as expected
-workspace_spm: aspects generate_xcodeproj
-	$(eval BUILD_DIR=$(shell xcodebuild -showBuildSettings \
-			-project XCHammer.xcodeproj/ \
-			-scheme XCHammer \
-			|  awk '$$1 == "BUILD_DIR" { print $$3 }'))
-	@mkdir -p "$(BUILD_DIR)"
-	@ditto "$(ASPECTDIR)" "$(BUILD_DIR)/Debug/TulsiGenerator.framework"
-	@ditto "$(ASSETDIR)" "$(BUILD_DIR)/Debug/$(ASSETDIR)"
-	@ditto "$(ASPECTDIR)" "$(BUILD_DIR)/Release/TulsiGenerator.framework"
-	@ditto "$(ASSETDIR)" "$(BUILD_DIR)/Release/$(ASSETDIR)"
-
-
 # Make an XCHammer XCHammer Xcode project.
-#
-# Note:
-# - this is under development and doesn't fully work
-# - incremental builds are currently not working with Bazel
-# - run with `force` for development
-# - the DSL is not fully integrated into XCHammer - this make target needs to
-#   create the XCHammer.yaml out of band.
-# - Currently gen times are out of band as well.
-workspace_xchammer: build
-	tools/bazelwrapper build :xchammer_config
+# Note: we manually build the XCHammer config here, consider better support to
+# build the project DSL within XCHammer directly.
+workspace: build
+	@tools/bazelwrapper build :xchammer_config
 	$(XCHAMMER_BIN) generate \
```

Comment:
you can actually use `Bazel run` here if  you don't want a separate build step

Reply:
Any way to add `xchammer_config` as a dep of `xchammer`?